sspl_test/functional_tests/v2/mock_data/controllers.txt

```diff
@@ -0,0 +1,688 @@
+{
```

Comment:
same for mock_data dir, maintain under sspl_test dir and rename that too realstor_ws_mock_data